### PR TITLE
Bump installed version of `<dark-mode-toggle>`

### PR DESCRIPTION
**Pull request by tomayac**
The new release fixes a bug with non-hover devices that led to the toggle leaving too much bottom margin.

### CODE REVIEW

File: src/_css/main.css [hunk 1]
```diff
@@ -425,8 +425,8 @@ img {
 }
 
 dark-mode-toggle {
-  --dark-mode-toggle-light-icon: url(/_css/img/sun.svg);
-  --dark-mode-toggle-dark-icon: url(/_css/img/moon.svg);
+  --dark-mode-toggle-light-icon: url(/_css/img/moon.svg);
+  --dark-mode-toggle-dark-icon: url(/_css/img/sun.svg);
```
Comment (mathiasbynens) [inline src/_css/main.css]: lolwhoops

Comment (tomayac) [reply src/_css/main.css]: Right? 

But in the end, it's this old discussion if a button should show the current state, or the state after pressing the button.

Comment (chicoxyzzy) [reply src/_css/main.css]: Previous version (before this change) looks more intuitive to me 🙊

Comment (tomayac) [reply src/_css/main.css]: If you look at the [demo](https://googlechromelabs.github.io/dark-mode-toggle/demo/index.html), the naming makes sense for the `switch`, but not so much for the `toggle` appearance. Would renaming to `--dark-mode-toggle-on-icon` and `--dark-mode-toggle-off-icon` make it any better?

Comment (chicoxyzzy) [reply src/_css/main.css]: Oh, it seems that I was looking at old version. What I see now at [v8.dev](https://v8.dev) LGTM 🙂.

**Pull request merged**